src/screens/OtherProfile/index.js

```diff
@@ -229,34 +118,6 @@ const OtherProfile = () => {
     }
   };
 
-  const sendSignedDM = async () => {
-    try {
-      setLoadingSendDM(true);
-      const signedMParams = {
-        user_id: dataMain.user_id,
-        message: dmChat
-      };
-      const response = await sendSignedDMOtherProfile(signedMParams);
-      const newResponse = {...response, members: response?.message?.members};
-      await saveChatFromOtherProfile(newResponse, 'sent', true, SIGNED);
-      setDMChat('');
-    } catch (error) {
-      if (__DEV__) {
-        console.log(error, 'error');
-      }
-    } finally {
-      setLoadingSendDM(false);
-    }
-  };
-
-  const onSendDM = async () => {
-    if (isAnonimity) {
-      await sentAnonDM();
-    } else {
-      await sendSignedDM();
-    }
-  };
-
   React.useEffect(() => {
     setOtherId(params?.data?.other_id);
   }, [params.data]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The useEffect hook that resets `dmChat` when `isSignedMessageEnabled` changes is removed. If it's necessary to reset some state when `isSignedMessageEnabled` changes, please ensure that this logic is handled elsewhere in your application.
```diff
- React.useEffect(() => {
-   setDMChat('');
- }, [isSignedMessageEnabled]);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
ok